CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ Slurm Ground Motion Workflow
 # Changelog
 (Based on https://wiki.canterbury.ac.nz/download/attachments/58458136/CodeVersioning_v18p2.pdf?version=1&modificationDate=1519269238437&api=v2 )
 
+## [19.7.11] - 2019-07-11 -- Verification plots added
```

Comment:
Read https://wiki.canterbury.ac.nz/download/attachments/58458136/CodeVersioning_v18p2.pdf?version=1&modificationDate=1519269238437&api=v2 to understand why we update major / minor version numbers. Let me know if that doesn't make sense or if you have any questions.

(p.s. you're gonna have another merge conflict in here too)